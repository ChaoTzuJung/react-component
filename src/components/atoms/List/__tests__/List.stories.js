import React from 'react';
import { storiesOf } from '@storybook/react';
import { withKnobs, object } from '@storybook/addon-knobs/react';

import withTests from 'util/storybook';

import List from 'components/atoms/List';

const items = [
	{
		key: 1,
		value: 'Lee',
	},
	{
		key: 2,
		value: 'Mike',
	},
];

const stories = storiesOf('atoms/List', module);

stories.addDecorator(withKnobs);
stories.addDecorator(withTests('List'));

stories.add('__interactive', () => <List items={object('list', items)} />);
