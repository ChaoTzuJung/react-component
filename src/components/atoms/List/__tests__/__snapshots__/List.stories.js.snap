// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<List
  items={
    Array [
      Object {
        "key": 1,
        "value": "Lee",
      },
      Object {
        "key": 2,
        "value": "Mike",
      },
    ]
  }
>
  <ul>
    <li
      key="1"
    >
      Lee
    </li>
    <li
      key="2"
    >
      Mike
    </li>
  </ul>
</List>
`;
